src/scenes/Event/components/TransactionHistory/index.js

```diff
@@ -46,7 +46,7 @@ export default class TransactionHistory extends Component {
             </TableHead>
             <TableBody>
               {transactions.map((transaction) => (
-                <TxRow key={transaction.txid} transaction={transaction} />
+                (!(this.props.my && transaction.senderAddress !== wallet.currentAddress)) && <TxRow key={transaction.txid} transaction={transaction} />
```

Comment:
this doesn't seem like it will show the All Transactions list?

Reply:
If param `this.props.my` set to false, the transaction will not filtered.
`<TransactionHistory my />` will show the filtered transactions and `<TransactionHistory />` will show all transactions.

Reply:
got it, you were only applying filtering for MyTransactions